pdp/handlers.go

```diff
@@ -113,6 +116,15 @@ func Routes(r *chi.Mux, p *PDPService) {
 
 	// PUT /pdp/piece/upload/{uploadUUID}
 	r.Put(path.Join(PDPRoutePath, "/piece/upload/{uploadUUID}"), p.handlePieceUpload)
+
+	// GET /pdp/piece/uploads
+	r.Get(path.Join(PDPRoutePath, "/piece/uploads"), p.handleGetStreamingUploadURL)
```

Comment:
This probably should be `POST` as it is not idempotent and creates a new location.

Reply:
Will switch to POST